src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/appsettings.json

```diff
@@ -9,7 +9,8 @@
         "AllowIdentityUpdateOverrides": "",
         "MaskRequestBodyInLogs": true,
         "UseLocalApiSchemaJson": false,
-        "MaximumPageSize": 500
+        "MaximumPageSize": 500,
+        "PathBase": ""
```

Comment:
Was the hardcoded "data" portion of the path removed? It should probably be defaulted here instead.